packages/tools/devtools/devtools-view/src/components/TelemetryView.tsx

```diff
@@ -192,7 +192,7 @@ export function TelemetryView(): React.ReactElement {
 					)}
 				</div>
 				<div>
-					<Button onClick={handleLoadMore} size="small">
+					<Button aria-label="Refresh Telemetry" onClick={handleLoadMore} size="small">
```

Comment:
For my knowledge: do you know if accessibility best practices would have us say "Refresh Telemetry Button" here? Or is that redundant given that the label is on a button element?

Reply:
Screen readers will announce the element type (in this case, "button") in addition to the aria label text, so including "Button" would be redundant.